### DISCUSSION TITLE
ScriptReference/Texture2D.LoadRawTextureData

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#213] [Notes]
# ScriptReference/Texture2D.LoadRawTextureData

https://docs.unity3d.com/ScriptReference/Texture2D.LoadRawTextureData.html

**Comment by mvi** [comment]
Note this works with raw pixel data, it does not work with image files. To load a PNG or JPG you should use [`LoadImage`](https://docs.unity3d.com/ScriptReference/Texture2D.LoadImage.html) instead.
